### ISSUE TITLE
ntpd should be stooped when chronyd is started in RHEL 7

### ISSUE THREAD

**Issue by pszypowicz** [thias/puppet-ntp#5]
Hi, Would you consider stopping ntpd when chronyd is mamaged in RHEL 7?

use case: Google Cloud engine has ntpd on rhel/centos 7 configured by default.

**Comment by robinbowes** [comment]
I agree this would be a useful capability.

I'm currently doing this in the class that instantiates the ntp class:
```
  # stop ntpd op CentOS 7
  if $::osfamily == 'Redhat' and $::operatingsystemmajrelease == '7' {
    service{'ntpd':
      ensure => stopped,
      enable => false,
    }->
    Class['::ntp']
  }
```

**Comment by thias** [comment]
Thanks for the feedback! This should be fixed in 9da371d1cdf55dc14f9c70d8e1f3d4927f0df90c.
